tools/swarm/src/compose.rs

```diff
@@ -757,7 +784,12 @@ mod tests {
                 init: true
                 command: |-
                   /bin/sh -c "
-                  kagami genesis sign /config/genesis.json --public-key $$GENESIS_PUBLIC_KEY --private-key 8026405A6D5F06A90D29AD906E2F6EA8B41B4EF187849D0D397081A4A15FFCBE71E7C73420F48A9EEB12513B8EB7DAF71979CE80A1013F5F341C10DCDA4F6AA19F97A9 --out-file $$GENESIS_SIGNED_FILE &&
+                  kagami genesis sign /config/genesis.json \\
+                      --public-key $$GENESIS_PUBLIC_KEY \\
```

Comment:
seems superfluous logically

Reply:
I'm saying this only as a reminder

Reply:
It is about newlines, right? They are not required, but when arguments are on separate lines, it is much easier to read or check generated docker compose files

Reply:
No, I was talking about the public key. Nvm